pkg/discovery/stitching/stitching_manager.go

```diff
@@ -30,6 +30,11 @@ const (
 
 	// The attribute used for stitching with other probes (e.g., prometurbo) with app and service
 	AppStitchingAttr string = "IP"
+
+	// The attribute used for stitching with Dynatrace probes with service
```

Comment:
"The attributes used for stitching kubeturbo discovered services with services discovered by the Dynatrace probes."

Reply:
will get this updated